docs/dev_guide/compare-prog-models.rst

```diff
@@ -0,0 +1,767 @@
+CUDA* and SYCL* Programing Model Comparison
+===========================================
+
+This section compares the CUDA* and SYCL* programming models and shows how to map
+concepts and APIs from CUDA to SYCL.
+
+Execution Model
+---------------
+
+Kernel Function
+***************
+
+In CUDA*, a kernel function is defined using the ``__global__`` declaration
+specifier and is executed concurrently across multiple threads on the GPU.
+Functions that are called by a CUDA kernel must be qualified with the ``__device__``
+specifier. The ``__host__`` declaration specifier is used to qualify functions
+that can be called from host code running on the CPU.
+
+In contrast, a SYCL* kernel is a function that is executed on SYCL-capable devices,
+such as CPUs, GPUs, or FPGAs. These kernels are launched from host code and are
+executed concurrently on SYCL devices. Unlike CUDA, SYCL kernel functions do not
+require special declaration specifiers and are defined using standard C++ syntax.
+
+The following table shows CUDA and SYCL equivalencies for defining kernel functions:
+
+.. list-table::
+   :widths: 50 50
+   :header-rows: 1
+
+   * - CUDA
+     - SYCL
+   * - ``__global__ void foo_kernel()``
+     - ``{}	void foo_kernel() {}``
+   * - ``__device__ void foo_device()``
+     - ``{}	void foo_device() {}``

```

Comment:
line33 should be ``void foo_kernel() {}``
line35 should be ``void foo_device() {}``

Reply:
Great catch! Done.